src/lib/evas/meson.build

```diff
@@ -134,16 +141,25 @@ evas_include_directories += [
   include_directories('.'),
   include_directories('common'),
   include_directories('include'),
-  include_directories('filters'),
 ]
 
+if get_option('ector')
+  evas_include_directories += [
+    include_directories('filters'),
+  ]
+endif
+
 evas_src += files([
    'main.c'
 ])
 
 evas_src_opt = [ ]
 
-evas_ext_none_static_deps += dependency('freetype2')
+if get_variable('freetype', false) == false
+  freetype = dependency('freetype2')
+endif
+
+evas_ext_none_static_deps += freetype
```

Comment:
What was the problem with FreeType?

Reply:
Tbh I find this strange as well. I think I applied an extra `git stash pop` by accident.

Reply:
Oooh right right, now I remember. This is sort of a fix to the `freetype` dependency. First, see the first appearence of `freetype` in `src/lib/ector/meson.build`:

```meson
ector_ext_deps = [triangulator, freetype, draw_without_deps, m]
```

It gets freetype from the static_lib, see the main `/meson.build`:

```meson
  ['ector'            ,['ector']             , false,  true, false,  false,   true,     false,  true, ['eina', 'efl'], ['draw', 'triangulator', 'freetype']],
```

Which defines the `freetype` variable and etc. What I did was to always get the variable when it exists, otherwise get from the system-wide dependency.

Reply:
Oh ok, make sense.
But wouldn't be better to do this at the `meson.build` at the root of the project?

Reply:
Oh, just noticed that `freetype` is declared at `src/static_libs/freetype/meson.build` so maybe there would be even better. What do you think?

Reply:
It could be, yes.